app/assets/stylesheets/components/_player.scss

```diff
@@ -105,8 +105,9 @@
     white-space: nowrap;
 }
 
-#player-volume {
-    margin:20px 15px 0 5px;
+#player-volume-slider-wrapper {
+    display: inline-block;
+    margin:19px 15px 0 5px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space